src/lib/error-messages.js

```diff
@@ -64,3 +64,24 @@ export const badSettingsInput = errors => {
 
   return 'Image settings must be positive integers.';
 };
+
+export const invalidBounds = errors => {
```

Comment:
Your validation doesn't seem to be working. When I "break" the bounds, I get a message in the console indicating as much (`Invalid bounds argument.`) but no feedback for the user.

Reply:
Done